docs/man-pages/htcondor.rst

```diff
@@ -3,84 +3,169 @@
 *htcondor*
 ===============
 
-Manage HTCondor jobs, job sets, and resources
+Manage HTCondor jobs, job sets, dags, logs and resources
 :index:`htcondor<single: htcondor; HTCondor commands>`\ :index:`htcondor command`
 
 Synopsis
 --------
 
 **htcondor** [ **-h** | **--help** ] [ **-v** | **-q** ]
 
-**htcondor** **job** [ *submit* | *status* | *resources* ]
-[**--resource** *resource-type*]
-[**--runtime** *time-seconds*]
-[**--email** *email-address*]
-[**--skip-history**]
+| **htcondor** **job** *submit* [**--resource** *resource-type*] [**--runtime** *time-seconds*] [**--email** *email-address*] submit_file
+| **htcondor** **job** *status* [**--resource** *resource-type*] [**--skip-history**] job_id
+| **htcondor** **job** *resources* [**--resource** *resource-type*] [**--skip-history**] job_id
 
 | **htcondor** **jobset** *submit* description-file
 | **htcondor** **jobset** *list* [**--allusers**]
-| **htcondor** **jobset** [ *status* | *remove* ] job-set-name [**--owner** *user-name*] [**--nobatch**] [**--skip-history**]
+| **htcondor** **jobset** *status* job-set-name [**--owner** *user-name*] [**--nobatch**] [**--skip-history**]
+| **htcondor** **jobset** *remove* job-set-name [**--owner** *user-name*] [**--nobatch**] [**--skip-history**]
 
 | **htcondor** **dag** *submit* dag-file
 | **htcondor** **dag** *status* dagman-job-id
 
+| **htcondor** **eventlog** *read* [ **-csv** | **-json**] [ **--groupby attribute** ] eventlog
+| **htcondor** **eventlog** *follow* [ **-csv** | **-json**] [ **--groupby attribute** ] eventlog 
+
 Description
 -----------
 
-*htcondor* is a tool for managing HTCondor jobs, job sets, and resources.
+*htcondor* is a tool for managing HTCondor jobs, job sets, resources, logs and
+dags.  It can replace *condor_submit*, *condor_submit_dag*, *condor_q*,
```

Comment:
```suggestion
DAGs.  It can replace *condor_submit*, *condor_submit_dag*, *condor_q*,
```
The colloquialism is appropriate for the command-line; not sure about for the documentation.